python/ml4ir/base/model/optimizers/optimizer.py

```diff
@@ -90,15 +83,15 @@ def choose_scheduler(model_config):
 
         if lr_schedule_key == LearningRateScheduleKey.EXPONENTIAL:
             learning_rate_schedule = ExponentialDecay(
-                initial_learning_rate=lr_schedule['learning_rate'] if 'learning_rate' in lr_schedule else 0.01,
-                decay_steps=lr_schedule['learning_rate_decay_steps'] if 'learning_rate_decay_steps' in lr_schedule else 100000,
-                decay_rate=lr_schedule['learning_rate_decay'] if 'learning_rate_decay' in lr_schedule else 0.96,
+                initial_learning_rate=lr_schedule['learning_rate'] if 'learning_rate' in lr_schedule else DefaultValues.CONSTANT_LR,
```

Comment:
Consider using the `get` method of a dictionary: 
```python
initial_learning_rate=lr_schedule.get('learning_rate', DefaultValues.CONSTANT_LR)
```
Same for the usecases below.

Reply:
+1

@balikasg You are beating me to all the comments during my PTO. Haha.